manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -1,23 +1,27 @@
 template: |
-  {{- if .Values.istio_cni.enabled }}
   metadata:
-    annotations:
+    annotations: {
```

Comment:
This change is pretty annoying. Basically, if we have `annotations: ` due to none of the below `ifs` executing, then we will wipe out the users annotations. So we need `annotations: {}`, which requires trailing commas.

Alternative is to set up some map of the annotations then do `if len annotations == 0 { {} } else {print all annotations}` but that seemed even uglier